### PR TITLE
Add `useFallback` option

### LINKED ISSUE #100: Add replacement method for automated tests
In automated tests, it is often a problem to have different outputs on Windows and Unix. In order to compare outputs, one would usually normalize outputs. 

When using this module, for example, one would replace all `✔` (Unix) to `√` (Windows), or the other way around.

This module provides with a `replaceSymbols(string): string` method, but it is not meant for this purpose.

https://github.com/sindresorhus/figures/blob/b10ba989a9dd359faf0f197e4081b144d2e72931/index.js#L283-L294

It would be nice to have another method (for example called `forceFallbackSymbols(string): string`, basically the same except without the initial `shouldUseMain` check. This could be used in automated tests.

What do you think?

### PR DESCRIPTION
**Pull request by ehmicky**
Fixes #100.

### CODE REVIEW

**Comment by sindresorhus** [comment]
What's the thinking of having it as a separate method rather than an option for `replaceSymbols()`? Just curious.

**Comment by ehmicky** [comment]
Not thinking about it! That's a better idea. :)

I updated the PR to use a `useFallback` boolean option instead. Please let me know if the name sounds good.

File: readme.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ console.log(fallbackSymbols.tick);
 console.log(replaceSymbols('✔︎ check'));
 // On terminals with Unicode symbols:  ✔︎ check
 // On other terminals:                 √ check
+
+console.log(replaceSymbols('✔︎ check', {useFallback: true}));
+// On terminals with Unicode symbols:  √︎ check
+// On other terminals:                 √ check
```
Comment (sindresorhus) [inline readme.md:38]: Shouldn't these two show the same symbol since it's using the fallback no matter what?

File: readme.md [hunk 2]
```diff
@@ -32,6 +32,10 @@ console.log(fallbackSymbols.tick);
 console.log(replaceSymbols('✔︎ check'));
 // On terminals with Unicode symbols:  ✔︎ check
 // On other terminals:                 √ check
+
+console.log(replaceSymbols('✔︎ check', {useFallback: true}));
```
Comment (sindresorhus) [inline readme.md:36]: This is not a common need, so I prefer to move the example to where the option is defined in the docs.

File: readme.md [hunk 3]
```diff
@@ -50,9 +54,9 @@ Symbols to use when the terminal supports Unicode symbols.
 
 Symbols to use when the terminal does not support Unicode symbols.
 
-### replaceSymbols(string)
+### replaceSymbols(string, options?)
 
-Returns the input with replaced fallback Unicode symbols on older terminals.
+Returns the input with replaced fallback symbols, if the terminal has poor Unicode support.
```
Comment (sindresorhus) [inline readme.md:59]:
```suggestion
Returns the input with replaced fallback symbols if the terminal has poor Unicode support.
```

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: readme.md (hunk 2 above)
Comment (ehmicky) [reply readme.md]: Done in 0f42c12.

File: readme.md (hunk 1 above)
Comment (ehmicky) [reply readme.md]: Those two lines use the same symbol (the fallback one).

**Comment by sindresorhus** [comment]
The symbols look different to me:

Screenshot 2024-03-04 at 12 42 07

**Comment by ehmicky** [comment]
Are those the ones in the main example section? Those should be different.

The ones in the example section for `useFallback` use the same Unicode codepoint in my text editor (`U-221a`).

Screenshot from 2024-03-04 06-29-29

**Comment by sindresorhus** [comment]
This is what I see:

Screenshot 2024-03-04 at 13 48 41

**Comment by ehmicky** [comment]
Wow, this challenged my sanity but I finally figured it out.

The first symbol was a single character with two codepoints: `U+221a` (tick) followed by `U+FE0E` ([variation selection 15](http://www.unicode-symbol.com/u/FE0E.html)). Since the variation selector changed the aspect of the tick, both codepoints formed a single character.

On VSCode, when I looked up the unicode codepoint, it only showed me the first codepoint. But I used `hexdump` and saw the actual UTF-8 encoded codepoints.

I checked the other files, and it only happened in this example, and its copy in `index.test-d.ts`. I am not sure how I ended up inputting that variation selection 15, was this either VSCode or vim did? :thinking: 

So this is now fixed in b051c4e611efaca4b7b6a380524b06170d267a4e. :magic_wand:

**Comment by ehmicky** [comment]
Fixed merged conflict.

**Pull request merged**